valhalla/baldr/time_info.h

```diff
@@ -41,6 +41,14 @@ struct TimeInfo {
   // a timezone offset cache because doing the offset math is expensive
   baldr::DateTime::tz_sys_info_cache_t* tz_cache;
 
+  /**
+   * Create TimeInfo object with default parameters.
+   * @return    TimeInfo structure
+   */
+  static TimeInfo empty() {
```

Comment:
nitpick, id call this `invalid` because this disables any adjustments to the time.

Reply:
ok, will rename